examples/nextjs-with-typescript/pages/_document.tsx

```diff
@@ -1,21 +1,17 @@
 import * as React from 'react';
 import Document, { Html, Head, Main, NextScript } from 'next/document';
 import createEmotionServer from '@emotion/server/create-instance';
-import theme from '../src/theme';
+import theme, { roboto } from '../src/theme';
 import createEmotionCache from '../src/createEmotionCache';
 
 export default class MyDocument extends Document {
   render() {
     return (
-      <Html lang="en">
+      <Html lang="en" className={roboto.className}>
```

Comment:
This is following Next.js documentation https://nextjs.org/docs/basic-features/font-optimization
I believe setting in on the body would have the exact same behaviour but might cause confusion if this example is not following the same practices as Next.js does in the documentation

Reply:
Perfect, thanks for linking the docs 👍

Reply:
@PetroSilenius The linked Next.js docs show setting the `className` on root of `_app` file tho. Am I missing something?

Reply:
Hey previously it showed how it can be used with the `app` directory and there it was placed in the root layout to the `html` element. I've personally set up a few projects setting it to `html` element in `_document` and it has worked fine

The change was done here https://github.com/vercel/next.js/pull/42266
The `html` element is still used in the beta docs https://beta.nextjs.org/docs/optimizing/fonts